packages/nimbus/src/components/form-field/form-field.recipe.tsx

```diff
@@ -49,6 +49,8 @@ export const formFieldRecipe = defineSlotRecipe({
       color: "critical.11",
       fontSize: "var(--form-field-font-size)",
       lineHeight: "var(--form-field-line-height)",
+      display: "flex",
+      gap: "100",
```

Comment:
@misama-ct Wanted to let you know about a change I had to make to the FormFieldError component. The text was being displayed on a newline below the error icon. Needed to make this change so that all children could be displayed inline, next to the icon instead.